filebeat/docs/autodiscover-hints.asciidoc

```diff
@@ -1,4 +1,56 @@
-Filebeat supports a autodiscover based on hints from the provider:
+{beatname_uc} supports autodiscover based on hints from the provider. The hints system looks for
+hints in Kubernetes Pod annotations or Docker labels that have the prefix `co.elastic.logs`, as soon as
+the container starts, {beatname_uc} will check if it contains any hints and launch the proper config for
+it. Hints tell {beatname_uc} how to get logs for the given container. By default logs will be retrieved
+from the container using the `docker` input, you can use hints to modify this behavior. This is the full
```

Comment:
Comma splice. You can read about comma splices here: https://www.grammarly.com/blog/comma-splice/

Change to: "...input. You can use..."